stylesheets/blue/components/_button_primary.scss

```diff
@@ -4,6 +4,7 @@
 //
 // default - Normal state
 // .ButtonPrimary--fullWidth - A button that takes the whole width of its parent
+// .ButtonPrimary--brightTurquoise - A button that uses the brightTurquoise color
```

Comment:
Let's make a call tomorrow then

> On Aug 18, 2016, at 23:42, Bruno Azevedo <notifications@github.com> wrote:
> 
> In stylesheets/blue/components/_button_primary.scss:
> 
> > @@ -4,6 +4,7 @@
> >  //
> >  // default - Normal state
> >  // .ButtonPrimary--fullWidth - A button that takes the whole width of its parent
> > +// .ButtonPrimary--brightTurquoise - A button that uses the brightTurquoise color
> Buttons are one of the most repeated patterns in all of our apps. They always follow the same pattern. So I don't get how you can apply that reasoning to this particular component.
> I don't have enough context to support your lack of "enough information", even though it usually requires asking the design team, so do what you feel is right.
> 
> "And there have been so many redesigns that I don't think the design system is clear at the moment". From personal experience that's not true, you're just assuming things.
> 
> I get that there's some sort of pressure to redesign asap, but that shouldn't happen with our website. That's one project which shouldn't simply accept that kind of debt. Would a call tomorrow help you make a more supported decision? I'm really not ok with your assumptions and decision. Maybe we should take advantage of the situation and conceive some ideas to improve this lack of awareness of the design system state.
> By the way, have you run the Styleguide, it should help you get a better overview.
> (i'm writing this from my phone so it's harder to write properly so, sorry for any phrase harder to understand)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>